README.md

```diff
@@ -89,7 +101,28 @@ We also want to show some of the awesome animal sightings our traveler had. You'
 
 **YOUR NOTES**
 ```
-
+traveler.animalSightings.forEach(animal =>  {
+    const newSighting = document.createElement("li")
```

Comment:
It would be better to have the variable name reference the element you created, for ex: newSightingLi